### PR TITLE
PIL-945 Accessibility Statement added

### PR DESCRIPTION
**Pull request by javed-iqbal1**

### CODE REVIEW

File: conf/services/pillar2-frontend.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+serviceName: Report Pillar 2 top-up taxes
+serviceDescription: |
+  Report Pillar 2 top-up taxes is a digital service whereby groups with a turnover of €750m or more can register and subscribe to the tax service and enable compliance to the global tax policy.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /report-pillar2-top-up-taxes
+contactFrontendServiceId: pillar2-frontend
```
Comment (jassalrichy) [inline conf/services/pillar2-frontend.yml:6]: Can we check we have the correct value for contactFrontendServiceId ?

Comment (jassalrichy) [reply conf/services/pillar2-frontend.yml:6]: If your service is not already integrating with contact-frontend, we advise choosing an identifier that is specific to your service and unlikely to be used by any other service, **avoiding any special characters** or whitespace.

Comment (jassalrichy) [reply conf/services/pillar2-frontend.yml:6]: https://github.com/hmrc/accessibility-statement-frontend?tab=readme-ov-file#a-note-on-contactfrontendserviceid

Comment (jassalrichy) [reply conf/services/pillar2-frontend.yml:6]: @javed-iqbal1 has confirmed that other services also for a dash '-' for their contactFrontendServiceId.
On that basis I'll approve this PR.

**Review by jassalrichy — APPROVED** [review]

**Review by jassalrichy — APPROVED** [review]

**Review by AKKUM — APPROVED** [review]

**Pull request merged**